src/Universum.hs

```diff
@@ -74,13 +74,13 @@ import           Data.Traversable         as X (Traversable (..), fmapDefault,
 
 #if ( __GLASGOW_HASKELL__ >= 800 )
 import           Data.List.NonEmpty       as X (NonEmpty (..), nonEmpty)
-import           Data.Monoid              as X
-import           Data.Semigroup           as X (Option (..), Semigroup (sconcat, stimes),
+import           Data.Monoid              as X hiding ((<>))
+import           Data.Semigroup           as X (Option (..), Semigroup (sconcat, stimes, (<>)),
                                                 WrappedMonoid, cycle1, mtimesDefault,
                                                 stimesIdempotent, stimesIdempotentMonoid,
                                                 stimesMonoid)
 #else
-import           Data.Monoid              as X
+import           Data.Monoid              as X hiding ((<>))
```

Comment:
Somebody might say that if GHC's version is < 8 then we want to use `<>` from `Data.Monoid` module. But I think that at some point in future (maybe in relatively near future) we will drop support for GHC < 8. And all our projects are already using GHC >= 8. So I don't think there's much point to argue about this particular line.